src/sentry/migrations/0348_issue_alert_fallback.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 2.2.28 on 2023-01-04 19:14
+
+import logging
+
+from django.db import migrations, transaction
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def set_issue_alert_fallback(rule, fallthrough_choice):
+    actions = rule.data.get("actions", [])
+    new_actions = []
+    for action in actions:
+        id = action.get("id")
+        target_type = action.get("targetType")
+        if id == "sentry.mail.actions.NotifyEmailAction" and target_type == "IssueOwners":
+            action.update({"fallthroughType": fallthrough_choice})
+            new_actions.append(action)
+
+    rule.save()
+
+
+def migrate_project_ownership_to_issue_alert_fallback(project, ProjectOwnership, Rule):
+    with transaction.atomic():
+        # Determine whether this project has a fallback setting.
+        fallthrough_choice = None
+        try:
+            ownership = ProjectOwnership.objects.get(project=project)
+            fallthrough_choice = "AllMembers" if ownership and ownership.fallthrough else "NoOne"
+        except ProjectOwnership.DoesNotExist:
+            # Projects without fallbacks will be assigned the new "ActiveMembers" default.
+            fallthrough_choice = "ActiveMembers"
+
+        # We only migrate rules that are not pending deletion.
+        for rule in Rule.objects.filter(project=project, status=0):
+            set_issue_alert_fallback(rule, fallthrough_choice)
+
+        project.save()
+
+
+def migrate_to_issue_alert_fallback(apps, schema_editor):
+    Project = apps.get_model("sentry", "Project")
+    ProjectOwnership = apps.get_model("sentry", "ProjectOwnership")
+    Organization = apps.get_model("sentry", "Organization")
+    Rule = apps.get_model("sentry", "Rule")
+
+    # We migrate a project at a time, but we prefer to group by org so that for the
+    # most part an org will see the changes all at once.
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.filter(status=0)):
```

Comment:
Could you change `status=0` for OrganizationStatus.ACTIVE? (If that is what is looking to do)
https://github.com/getsentry/sentry/blob/32857908366610f57acd62a0cdd9acf5365e5055/src/sentry/models/organization.py#L45